phys/module_mp_thompson.F

```diff
@@ -4275,11 +4275,11 @@ subroutine qr_acr_qg(NRHGtable)
 
         if (.not. is_hail_aware) then
 !. Distribute the same value across NRHG dimension
-         do n3 = 2, NRHG
+         do n3 = 2, dimNRHG
```

Comment:
@mefrediani If dimNRHG = 1, would it work? Maybe you could check the value as a condition to do this loop?

Reply:
@mefrediani On second thought, perhaps this loop is no longer needed. What do you think?

Reply:
Good catch, Wei! I can't believe I missed this!

Reply:
Yes, probably no longer needed. Looks good so far. Hopefully this will solve the problem.